estela-api/core/migrations/0033_auto_20230814_2238.py

```diff
@@ -1,35 +0,0 @@
-# Generated by Django 3.1.14 on 2023-08-14 22:38
-
-import django.db.models.deletion
-from django.db import migrations, models
-
-
-class Migration(migrations.Migration):
-    dependencies = [
-        ("core", "0032_project_framework"),
-    ]
-
-    operations = [
-        migrations.AddField(
-            model_name="spiderjobenvvar",
-            name="project",
-            field=models.ForeignKey(
-                help_text="Project pid.",
-                null=True,
-                on_delete=django.db.models.deletion.CASCADE,
-                related_name="env_vars",
-                to="core.project",
-            ),
-        ),
-        migrations.AddField(
-            model_name="spiderjobenvvar",
-            name="spider",
-            field=models.ForeignKey(
-                help_text="Spider sid.",
-                null=True,
-                on_delete=django.db.models.deletion.CASCADE,
-                related_name="env_vars",
-                to="core.spider",
-            ),
-        ),
-    ]
```

Comment:
We shouldn't be deleting this migrations file. We also need to generate new migrations for the Proxy model.